13NLP.tex

```diff
@@ -30,7 +30,7 @@ \section{Character Features}
 \end{python}
 
 
-In \cite{sapkota2015not} have been identified 10 character n-grams categories, being proven as the most successful feature in both single-domain and cross-domain Authorship Attribution.\\
+In \cite{sapkota2015not}  10 character n-grams categorieshave been identified, being proven as the most successful feature in both single-domain and cross-domain Authorship Attribution.\\
```

Comment:
there's 2 spaces between \cite{sapkota2015not} and 10. and no space between categories and have